src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -84,7 +94,19 @@ public static AddressBook getTypicalAddressBook() {
         return ab;
     }
 
+    public static AddressBook getAddressBookWithOneFreshmanOrOgl() {
```

Comment:
should be "oneFreshmanAndOgl". Other than that, good use of test data

Reply:
Okay just updated!